### PR TITLE
image pull secrets doc

### PR DESCRIPTION
**Pull request by allenhaozi**
Signed-off-by: mahao <allenhaozi@gmail.com>

### Ⅰ. Describe what this PR does
image pull secrets doc

### CODE REVIEW

File: docs/en/samples/image_pull_secrets.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# image pull secrets
+
+
+In private scenario, maybe you need to set image pull secret.
```
Comment (cheyang) [inline docs/en/samples/image_pull_secrets.md:4]: If the images of fluid runtime is in private docker registry,  it's necessary to set image pull secret.

Comment (allenhaozi) [reply docs/en/samples/image_pull_secrets.md]: Thank you for your suggestion, done

**Review by cheyang — APPROVED** [review]

**Comment by cheyang** [comment]
thanks @allenhaozi , would you please add doc in Chinese as well?

**Comment by allenhaozi** [comment]
> thanks @allenhaozi , would you please add doc in Chinese as well?

No problem, I will submit a pr in Chinese

**Pull request merged**